### PR TITLE
LRQA-70572 | master

### PR DESCRIPTION
**Pull request by 91nilo**
https://issues.liferay.com/browse/LRQA-70572

### CODE REVIEW

**Comment by 91nilo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase [hunk 1]
```diff
@@ -517,28 +517,22 @@ definition {
 	@description = "This is a use case for LPS-71236 and LPS-71307. View web content."
 	@priority = "5"
 	test NoXSSVulnerabilityInWebContent {
-		task ("Add CP") {
+		task ("Given: a Web Content with a script in the title is added to the display page") {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase:520]: Prefer to break up into two task groups like:
Given: Create a web content with a script in the title.
And: Select web content display in WCD.

**Comment by 91nilo** [comment]
Hi @JoyceWang08211, I've applied the changes, if there's something else to improve, please tell me.
Thanks!

**Comment by 91nilo** [comment]
ci:test:sf

**Comment by JoyceWang08211** [comment]
LGTM

**Comment by 91nilo** [comment]
ci:forward

**Pull request closed without merging**